WalletWasabi.Fluent/ViewModels/Wallets/WalletRenameViewModel.cs

```diff
@@ -0,0 +1,50 @@
+using System.Reactive;
+using ReactiveUI;
+using WalletWasabi.Fluent.Models.Wallets;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+[NavigationMetaData(Title = "Rename Wallet")]
+public partial class WalletRenameViewModel : DialogViewModelBase<Unit>
```

Comment:
Use `RoutableViewModel` here. 

`DialogViewModelBase<>` is intended for scenarios where dialogs return a value to their caller. This is not the case as this is retuning `Unit`

Reply:
That's OK. The problem is that semantics change a bit in this case. We lost the Close method and we need to Navigate().Back(), that feels like a hack to me.

Reply:
Done BTW :)